### DISCUSSION TITLE
In the sample configuration, what is the purpose of 'resolveAdditionalTextEditsSupport'?

### DISCUSSION THREAD

**Discussion by DasOhmoff** [mfussenegger/nvim-jdtls#356] [Q&A] [answered]
Hey,

I was looking through the sample configuration of [@cjgratacos](https://github.com/cjgratacos), and I save the following lines of code:
```lua
    local extendedClientCapabilities = require'jdtls'.extendedClientCapabilities
    extendedClientCapabilities.resolveAdditionalTextEditsSupport = true
    config.init_options = {
      -- bundles = bundles;
      extendedClientCapabilities = extendedClientCapabilities;
    }
```
I was wondering what the porpose of this code is. What does `resolveAdditionalTextEditsSupport` do? I was not able to find any documentation in the whole internet in regards to this.

Thank you for your help :)

**Comment by mfussenegger** [comment, marked as answer]
It tells the server that the client supports resolving `additionalTextEdits` lazy in completion items. One should only set that if using a plugin that actually supports it.

See https://github.com/eclipse/eclipse.jdt.ls/pull/1487 for more context

**Comment by DasOhmoff** [comment]
Ah OK. Thanks, that is exactly what I wanted to know!
